rest/src/main/scala/io/udash/rest/raw/AbstractMapping.scala

```diff
@@ -0,0 +1,109 @@
+package io.udash
+package rest.raw
+
+import com.avsystem.commons._
+import io.udash.rest.raw.AbstractMapping.ConcatIterable
+
+import scala.collection.generic.CanBuildFrom
+import scala.collection.mutable
+
+sealed abstract class AbstractMapping[V] extends PartialFunction[String, V] {
```

Comment:
Probably needs at least a small comment on why we need yet another collection for a kv list

Reply:
There are docs on subclasses, you think it's enough?

Reply:
As long as you don't use `AbstractMapping` in user API it's fine.